.circleci/config.yml

```diff
@@ -14,63 +15,134 @@ jobs:
       - TOOL_NODE_FLAGS: "--max-old-space-size=4096"
 
     steps:
-      - setup_remote_docker
       - checkout
 
-      # install OS dependencies
       - restore_cache:
           name: Restoring Meteor cache
-          key: meteor
+          key: meteor-{{ .Branch }}
```

Comment:
Added branch name to the keys for the cache primarily because the old cache that was generated on previous node:8 image is to be linked/restored to root locations on the new 8.9.3 (which also resulted in permission denied errors).